vqfx/Makefile

```diff
@@ -19,17 +19,8 @@ format-legacy-images:
 
 # TODO: we should make sure we only copy one PFE image (the latest?), in case there are many
 docker-pre-build:
-
 	echo "pfe     $(PFE_IMAGE)"
 	cp vqfx*-pfe*.qcow* docker/
 	echo "image   $(IMAGE)"
 	echo "version $(VERSION)"
 
-# mostly copied from makefile.include, i wish this was easier to change
-docker-build-common: docker-clean-build docker-pre-build
-	@if [ -z "$$IMAGE" ]; then echo "ERROR: No IMAGE specified"; exit 1; fi
-	@if [ "$(IMAGE)" = "$(VERSION)" ]; then echo "ERROR: Incorrect version string ($(IMAGE)). The regexp for extracting version information is likely incorrect, check the regexp in the Makefile or open an issue at https://github.com/hellt/vrnetlab/issues/new including the image file name you are using."; exit 1; fi
-	@echo "Building docker image using $(IMAGE) as $(REGISTRY)vr-$(VR_NAME):$(VERSION)"
-	cp ../common/* docker/
-	$(MAKE) IMAGE=$$IMAGE docker-build-image-copy
-	(cd docker; docker build --build-arg http_proxy=$(http_proxy) --build-arg https_proxy=$(https_proxy) --build-arg RE_IMAGE=$(IMAGE) --build-arg PFE_IMAGE=$(PFE_IMAGE) -t $(REGISTRY)vr-$(VR_NAME):$(VERSION) .)
```

Comment:
Hi @Zegroj 

I had a look at the makefile and I wonder if it is time to change it to the common workflow. I pushed over the change, if you do a pull and rerun `make` I wonder if you will build a bootable system. Thnx

Reply:
Hi @hellt i re run it with the change you made but didnt allow me to build a bootable system in the vqfx, i tried again with the old version but with this change "the variable NAME to VR_NAME" and worked fine

Reply:
ah, I see, I think the PFE image was missing when I removed this overriding build step. I will have a look